libsolidity/parsing/Parser.cpp

```diff
@@ -961,6 +961,14 @@ ASTPointer<Identifier> Parser::parseIdentifier()
 	return nodeFactory.createNode<Identifier>(expectIdentifierToken());
 }
 
+ASTPointer<Identifier> Parser::parseIdentifierIncludingAddress()
```

Comment:
Why not `IdentifierOrAddress` given that is what's used the scanner/ast? If the parser uses `including` more frequently, then of course that is a better choice.